VT_Domain_Scanner_py3.py

```diff
@@ -151,10 +153,8 @@ def DomainReportReader(domain, delay):
             try:
                 delay = DomainScanner(domain)
                 data = DomainReportReader(domain, delay)
-                with open('results.csv', 'a') as rfile:
-                    dataWriter = csv.writer(rfile, delimiter = ',')
-                    dataWriter.writerow(data)
-                    time.sleep(15)  # wait for VT API rate limiting
+                dataWriter.writerow(data)
+                time.sleep(15)  # wait for VT API rate limiting
```

Comment:
Looking back at this now I don't know why I would have left the file open for 15 seconds when trying to avoid the rate-limiting. Thanks for catching this!